examples/IRB140/runPlanning.m

```diff
@@ -1,14 +1,22 @@
-function runPlanning()
+function xtraj = runPlanning(q0, pos_final, visualize)
+% default starting pose
+if nargin<1
+  q0 = [0,0,0,0,0,0]';
+end
+%pos_final = [-0.5, 0, 1.015]';
+if (nargin<2)
+  pos_final = [-1.212+0.5, -0.1+0.5, 1.025]';
+end
+if (nargin<3)
+  visualize = false;
+end
 
 r=RigidBodyManipulator();
 r = addRobotFromURDF(r,'irb_140.urdf',[-0.5, 0, 1.5]',[pi/2, pi/2, 0]');
```

Comment:
i thought it was odd to have the IRB use this orientation.  Seems like we should orient it as it is on the lab table by default.